webpack/client.config.js

```diff
@@ -20,14 +22,26 @@ module.exports = {
 					options: {
 						dev: isDev,
 						hydratable: true,
-						hotReload: true
+						hotReload: true,
+						emitCss: true
 					}
 				}
+			},
+			{
+				test: /\.css$/,
+				use: [
+					isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
+					"css-loader"
+				]
 			}
 		]
 	},
 	mode,
 	plugins: [
+		new MiniCssExtractPlugin({
+			filename: "[name].css",
+			chunkFilename: "[id].css"
```

Comment:
The naming should probably match the sapper JS path and file name, so something like:
```
new MiniCssExtractPlugin({
      filename: '[hash]/[name].css',
      chunkFilename: '[hash]/[name].[id].css'
    }),
```

Reply:
There's also the issue of how to inject a reference to this file in the HTML template. I created an issue for that a while ago: https://github.com/sveltejs/sapper-template/issues/59